src/server/frontend_wayland/window_wl_surface_role.cpp

```diff
@@ -275,6 +276,18 @@ void mf::WindowWlSurfaceRole::set_fullscreen(std::experimental::optional<struct
     }
 }
 
+void mf::WindowWlSurfaceRole::set_server_side_decorated(bool server_side_decorated)
+{
+    if (weak_scene_surface.lock())
+    {
+        log_warning("server_side_decorated decorated property can not be changed after surface created");
```

Comment:
Why not?

AFAICS `zxdg_toplevel_decoration_v1.set_mode/unset_mode` can be called at any time.

Reply:
I haven't added the property to `SurfaceSpecification`, only `SurfaceCreationParamaters`. Such a property could be added in the future, but there isn't a need yet.

Reply:
In that case, can reword the log message as it is more a matter of "not implemented" than "not possible". (E.g. adding "yet".)